tools/create-cs.sh

```diff
@@ -0,0 +1,20 @@
+ #!/bin/bash
```

Comment:
Can we name this script `create-catalog-source.sh`?

Reply:
Also, can we use `#!/bin/sh`?  I don't see anything Bash-specific.

Reply:
Changed name. 
I left /bin/bash since it's the most common.